.env.development

```diff
@@ -8,4 +8,5 @@ REACT_APP_COIN_EXCHANGE_URL=https://rest.coinapi.io/v1/exchangerate/ETH/USD
 REACT_APP_FEATURE_AUTO_SELECT_OPERATORS=
 REACT_APP_SHOULD_CALCULATE_DOLLAR=
 REACT_APP_CONTRACT_ADDRESS=0x9573C41F0Ed8B72f3bD6A9bA6E3e15426A0aa65B
-REACT_APP_ETHER_SCAN_URL=https://goerli.etherscan.io/address/
\ No newline at end of file
+REACT_APP_ETHER_SCAN_URL=https://goerli.etherscan.io/address/
+REACT_APP_EXPLORER=https://explorer.stage.ssv.network
```

Comment:
`REACT_APP_EXPLORER` meaningless, it should be at least `REACT_APP_EXPLORER_URL`